### ISSUE TITLE
Recipient region and country conflicting

### ISSUE THREAD

**Issue by danmackenzie** [younginnovations/aidstream-v201-migration#15]
A common hurdle to publishing 2.01 XML in Aidstream is that 2.01 require either Recipient Country OR Recipient Region. It does not like both. 

Not a bug necessarily, but an easy thing to raise with users.

**Comment by anjesh** [comment]
http://support.iatistandard.org/entries/52502089-Upgrading-to-2-01-What-s-involved-A-non-technical-summary says either Recipient Country or Recipient Region, so that validation stuff in place. 
Thoughts? @iatisupport

**Comment by rolfkleef** [comment]
I don't read it as either-or, but at least one of them needs be there.

There's even an example of both of them included with the text "When both the recipient-region and recipient-country, then the @percentage values should sum to 100% for the specific iati-activity."

http://iatistandard.org/201/activity-standard/iati-activities/iati-activity/recipient-country/

**Comment by iatisupport** [comment]
Yes @rolfkleef is correct in that country and region are allowed together but only when the @percentage values are used
